Extensions/ArtifactEngine/EngineTests/artifactEngineTests.ts

```diff
@@ -10,7 +11,7 @@ mockery.enable({
 mockery.registerMock('fs', {
     createWriteStream: (a) => {
         var mockedStream = stream.Writable();
-        mockedStream._write = () => {};
+        mockedStream._write = () => { };
```

Comment:
does the additional space matter?

Reply:
no, but when i autoformat in vscode it generates this space.